db/migrate/20171001035221_move_setting_to_tags.rb

```diff
@@ -0,0 +1,23 @@
+class MoveSettingToTags < ActiveRecord::Migration[5.0]
+  def up
+    print "\n"
+    Character.where.not(setting: nil).each do |character|
+      setting = Setting.where(name: character.setting).first
+      setting ||= Setting.create!(user: character.user, name: character.setting)
```

Comment:
Can we cache settings by name here for efficiency purposes? (This is looking to be a lot of `where`s)